### ISSUE TITLE
Path2D curves problem with point

### ISSUE THREAD

**Issue by Torguen** [godotengine/godot#52629]
### Godot version

3.3.3 stable mono

### System information

w10 64

### Issue description

When a point is removed and the opposite is moved, a new point is created again.
Godot allows a point to be removed, but for what?
A new point should not be created when I deleted the old one.
Everything happens with the tool selected in the capture.

![Captura3](https://user-images.githubusercontent.com/19652075/133073327-00dd6383-21bb-4e7e-b555-94a0c7e00857.PNG)

![gif gif](https://user-images.githubusercontent.com/19652075/133073026-7b0eed5d-f3f1-41c7-92c4-c628286e9e67.gif)

### Steps to reproduce

See the gif please

### Minimal reproduction project

...

**Comment by kleonc** [comment]
Turning off any of the mirroring options:
![Godot_v3 4-beta4_win64_B6LlmVM4ts](https://user-images.githubusercontent.com/9283098/133076726-8b55991d-0f7b-49e7-a823-c178dcba7643.png)
will make it work as you expect.

**Comment by Torguen** [comment]
I'm not sure about that. I mean if I delete the point then godot shouldn't mirror and create a new point.
By default 2 points are created but if the user deletes a point then godot should not create it even if the mirroring option is activated. That seems to me.
Thanks for the information.

**Comment by kleonc** [comment]
Yeah, it kinda makes sense for it to behave by default the way you're suggesting. If someone would want the other handle to be added again it can be done by dragging from the point again.

Besides what you're suggesting there's a problem that these mirror flags aren't independent from each other. You can't make handles mirror only the length:
![TsZrfgPa41](https://user-images.githubusercontent.com/9283098/133085014-a89f8aac-a9cd-4806-bf7f-9d929ebe07a7.gif)
